packages/reason-parser/reason-parser.2.0.0/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "1.2"
+maintainer: "Jordan Walke <jordojw@gmail.com>"
+authors: "Jordan Walke <jordojw@gmail.com>"
+homepage: "https://github.com/facebook/reason"
+bug-reports: "https://github.com/facebook/reason/issues"
+license: "BSD"
```

Comment:
The idea was at certain point to standardize on [spdx](https://spdx.org/licenses/) for the license field. I'm not sure if what facebook uses is expressible there but in any case just not having `BSD` in there but `BSD-3-Clause+facebook` would be sufficient to trigger further investigations.